test/anacondalib.py

```diff
@@ -50,6 +56,17 @@ def setUp(self):
 
         super().setUp()
 
+        # Add installation target disk
+        backing_file = None if not self.disk_image else os.path.join(BOTS_DIR, f"./images/{self.disk_image}")
+        self.add_disk(15, backing_file)
```

Comment:
that's 15 GB? (passed to `--disk`). What does that mean with an overlay? My gut feeling is that the size just gets ignored then?

Reply:
You re right, adjusted.

Reply:
It was just a question, it looks a bit strange now -- but I'll comment inline.

Reply:
Actually it's fine, it just looked confusing in the interdiff.